include/events/signal.hpp

```diff
@@ -110,6 +110,9 @@ namespace signals {
     struct tray_pos_change : public detail::value_signal<tray_pos_change, int> {
       using base_type::base_type;
     };
+    struct tray_hide : public detail::value_signal<tray_hide, bool> {
```

Comment:
Maybe call it `tray_visibility`. `tray_hide` is a bit confusing, especially since `false` means hide and `true` means show.